packages/stack-link/README.md

```
@@ -1 +1,603 @@
+
+  return <NormalMode />;
+}
+```
+
+## 작동 원리
+
+### 페이지 구조
+
+StackLinkProvider는 다음과 같은 3개의 주요 영역을 생성합니다:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)